cross/he853/PLIST

```diff
@@ -0,0 +1,2 @@
+bin:bin/he853
+lib:lib/udev/rules.d/80-he853.rules
```

Comment:
Done

Nearby `BUILD_DEPENDS` should be fine since libusb-1.0 is statically linked in the executable. I wanted to avoid any clashing with other stuff using libusb-1.0 (though I couldn't find anything which uses it anyway)